tox.ini

```diff
@@ -9,8 +9,9 @@ envlist =
     pylint
     # Python/Django combinations that are officially supported
     py3{6,7,8,9,10}-django32
-    py3{8,9,10,11}-django41
-    py3{8,9,10,11}-django42
+    py3{8,9,10,11,12}-django41
+    py3{8,9,10,11,12}-django42
+    py3{8,9,10,11,12}-django50
```

Comment:
Django 5.0 requires Python >= 3.10 according to the [package metadata on PyPI](https://pypi.org/project/Django/).

After adjusting this line you also need to exclude Python < 3.10 for Django 5.0 in the [test matrix of GHA](https://github.com/behave/behave-django/blob/main/.github/workflows/test.yml#L32-L37).

Reply:
Thanks. I have made [relevant changes](https://github.com/behave/behave-django/pull/153/files).